Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/CertificateStateHelper.kt

```diff
@@ -73,36 +74,48 @@ fun IncludeCertificateQrcodeCardBinding.bindValidityViews(
             expirationStatusBody.isVisible = isCertificateDetails
             expirationStatusBody.text = context.getText(R.string.expiration_info)
             startValidationCheckButton.isVisible = isPersonDetails
+            certificateBadgeCount.isVisible = badgeCount != 0
+            certificateBadgeCount.text = badgeCount.toString()
+            certificateBadgeText.isVisible = badgeCount != 0
```

Comment:
I think if you moved outside of when it should be enough , it is already the same lines in every case , can you try that ?

Reply:
Yes, good idea.